### ISSUE TITLE
quadbinZoom error

### ISSUE THREAD

**Issue by menusal** [CartoDB/carto-react#527]
Version of C4R ^1.4.5

After the last update of the `C4R` version and working with spatial indexes layers, in this case, a `quadbin` layer, the `quadbin` library seems than isn't imported correctly. 

## Error

```
Uncaught TypeError: (0 , A._quadbinZoom) is not a function
    at 6676cb64-5f27-45d4-bd28-0d8487d404eb:2:231929
    at $A (6676cb64-5f27-45d4-bd28-0d8487d404eb:2:231989)
    at te (6676cb64-5f27-45d4-bd28-0d8487d404eb:2:233793)
    at 6676cb64-5f27-45d4-bd28-0d8487d404eb:2:243324
    at onmessage (6676cb64-5f27-45d4-bd28-0d8487d404eb:2:243445)
```

<img width="1436" alt="image" src="https://user-images.githubusercontent.com/5357085/201056138-47bc91e8-9b22-492b-acbf-93c74688d653.png">

<img width="1438" alt="image" src="https://user-images.githubusercontent.com/5357085/201059031-3a8e25fd-31b5-43a2-86e6-42a9450dd96f.png">

## Code and intructions to reproduce the unexpected behavior

- Clone this repo and install with C4R instructions: https://github.com/menusal/test-c4r.
- Go to https://127.0.0.1:3000/TestGrid.
- Watch the console.

You must sign in with `ps-cpg+dev@cartodb.com`credentials. Ping me to provide +info

**Comment by Josmorsot** [comment]
@felixpalmer looks like `quadbinZoom` doesn't exists in `quadbin-js`. This function was exported in `@deck.gl/carto`.

**Comment by Josmorsot** [comment]
Fixed in v1.4.6
